### ISSUE TITLE
No Feedback from Screen Reader when Changing Visibility

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30875]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When you press V to change an elements visibility there is no speech feedback from the screen reader.
This should announce the state when it changes. ie It should say "Visible" or "Invisible".

A number of other issues were resolved from ticket 20041 in 4.6 but this still has no speech feedback.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.3

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
